app/views/projects/attachments/_delete_attachment_modal.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= viral_dialog(open: open, size: :large) do |dialog| %>
+  <%= dialog.with_header(title: t(".title")) %>
+  <%= dialog.with_section do %>
+
+    <%= turbo_frame_tag("deletion-alert") %>
+
+    <div
+      class="mb-4 font-normal text-slate-500 dark:text-slate-400 overflow-x-visible">
+      <p class="mb-4">
+        <%= t(".description") %>
+      </p>
+      <%= form_for(:deletion, url: namespace_project_attachment_path(id: @attachment.id), method: :delete) do |form| %>
+        <%= form.submit t(".submit_button"),
+                    class: "
+                      button
```

Comment:
**nitpick:** Don;t use those pre-defined classes, I am trying to get rid of them

Reply:
Replaced in 604de82cfeb991875d5ebf9263de9739a7610a6d